packages/core/onClickOutside/index.ts

```diff
@@ -1,10 +1,12 @@
-import { Fn, tryOnUnmounted } from '@vueuse/shared'
+import { tryOnUnmounted } from '@vueuse/shared'
 import { MaybeElementRef, unrefElement } from '../unrefElement'
 import { useEventListener } from '../useEventListener'
 import { ConfigurableWindow, defaultWindow } from '../_configurable'
 
-const events = ['mousedown', 'touchstart', 'pointerdown'] as const
-type EventType = WindowEventMap[(typeof events)[number]]
+export type AllowedEvents = Pick<WindowEventMap, 'mousedown' | 'mouseup' | 'touchstart' | 'touchend' | 'pointerdown' | 'pointerup'>
```

Comment:
```suggestion
export type OnClickOutsideAllowedEvents = Pick<WindowEventMap, 'mousedown' | 'mouseup' | 'touchstart' | 'touchend' | 'pointerdown' | 'pointerup'>
```

Reply:
I knew it 😂 What about `OnClickOutsideEvents`? We are using it a lot on code and this is little bit longer.

Reply:
Sure.